Java/src/test/java/AirportTest.java

```diff
@@ -74,32 +74,28 @@ public void test3() {
     }
 
     @Test
-    public void testHasAtLeastOneBomberInMilitaryPlanes() {
+    public void verifyHasAtLeastOneBomberInMilitaryPlanes() {
         Airport airport = new Airport(planes);
         List<MilitaryPlane> bomberMilitaryPlanes = airport.getBomberMilitaryPlanes();
-        boolean flag = false;
         for (MilitaryPlane militaryPlane : bomberMilitaryPlanes) {
             if ((militaryPlane.getType() == MilitaryType.BOMBER)) {
-                flag = true;
-            }
-            else {
+            } else {
                 Assert.fail("Test failed!");
             }
         }
-        // if not failed
     }
 
     @Test
-    public void testExperimentalPlanesHasClassificationLevelHigherThanUnclassified(){
+    public void verifyExperimentalPlanesHasClassificationLevelHigherThanUnclassified() {
```

Comment:
I'm not sure the Test name is correct because Assert.assertFalse(hasUnclassifiedPlanes)

Reply:
Fixed